site/src/contexts/auth/useAuth.ts

```diff
@@ -1,5 +1,5 @@
 import { useContext } from "react";
-import { AuthContext } from "./AuthProvider";
+import { AuthContext, type AuthContextValue } from "./AuthProvider";
 
 export const useAuth = () => {
```

Comment:
Wondering if it might be worth changing the name of this hook to indicate that we want people to favor `useAuthenticated` over it

Reply:
I think I'm going to make it `useAuthContext`